tsconfig.build.json

```diff
@@ -1,7 +1,13 @@
 {
-  "extends": "./tsconfig.json",
+  "extends": "@companion-module/tools/tsconfig/node14/recommended",
+  "include": ["src/**/*.ts"],
+  "exclude": ["node_modules/**", "src/**/*spec.ts", "src/**/__tests__/*", "src/**/__mocks__/*"],
   "compilerOptions": {
-    "types": ["node"]
-  },
-  "exclude": ["node_modules/**", "src/__tests__/**", "src/__mocks__/**"]
+    "module": "commonjs",
```

Comment:
I set the module format back to cjs here

Reply:
If you update `@companion-module/tools` that line will no longer be necessary